tests/Agent/IntegrationTests/UnboundedApplications/BasicMvcCoreApplication/Controllers/PostgresController.cs

```diff
@@ -18,6 +18,7 @@ public class PostgresController : Controller
     {
 
         [HttpGet]
+        [Route("Postgres/Postgres")]
```

Comment:
Should these be moved to ConsoleMF?

Reply:
Could be, but I prefer to work on this in a different story.